Classes/Plugins/Ads/PKAdInfo.swift

```diff
@@ -46,21 +47,6 @@ import GoogleInteractiveMediaAds
         }
     }
     
-    init(ad: IMAAd) {
```

Comment:
nope, this is because we need to import IMA to have this initializer. and when we moved the class to core we can't have IMA inside for sure.